mungegithub/mungers/fsm/prereview.go

```diff
@@ -28,7 +28,7 @@ const (
 	releaseNote               = "release-note"
 	releaseNoteActionRequired = "release-note-action-required"
 	releaseNoteExperimental   = "release-note-experimental"
-	releaseNoteLabelNeeded    = "release-note-none"
+	releaseNoteNone           = "release-note-none"
```

Comment:
Ah okay. Yeah, these are just the constant definitions, but `releaseNoteLabelNeeded` isn't called in this particular piece of code.